.packit.yaml

```diff
@@ -0,0 +1,58 @@
+# This file is used by the Packit service.
+# Currently it is used for the gating of pull requests.
+#
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+# We need a specfile to build the RPM package.
+# But instead of storing it in this repository,
+# we use the one that exists already in src.fedoraproject.org/rpms/golang.
+# In particular, we use the specfile from the branch `eln`.
+# We'll create the temporary folder .packit_rpm to store the content of the
+# Fedora repository.
+specfile_path: .packit_rpm/golang.spec
+
+# We need to tell Packit which files to sync from the upstream repository.
+files_to_sync:
+  - .packit.yaml
+  - ./scripts/packit.sh
```

Comment:
Do you need the changes from this script reflected in the final downstream rpm as well?

Reply:
You mean if they will be included in the CentOS Stream packages? They are only for this GitHub project, I don't think we need them anywhere else.

Reply:
In that case, line 18 can be removed too.